src/server.ts

```diff
@@ -1,3 +1,5 @@
+await import('./otel.ts')
```

Comment:
@kibertoad It's a learning opportunity for me - why do we need to import it if we aren't using anything from `otel.ts` here?" Do we want [this](https://github.com/lokalise/node-service-template/pull/1264/files#diff-08285840e6920c4912ba8fc22550788bce50675f267cf34a99f01e28cefb0d5aR8-R48) to be executed before `starting` the server? If yes, is there a way to make this intent more explicit?

Reply:
isn't line 6 of otel saying exsctly that? can you suggest a better wording?

Reply:
But I need to know that I should go to `otel.ts` to check this. Reading the `server.ts`, I won't learn that we are using any OpenTelemetry - the code in `server.ts` doesn’t indicate that `otel` is being initialized. This information is implicit in `await import('./otel.ts')`. It would be clearer to have a method in `server.ts` called `initializeOtel()`, but maybe that’s not possible.